conjur/init/init_controller.py

```diff
@@ -0,0 +1,137 @@
+# -*- coding: utf-8 -*-
+
+"""
+InitController module
```

Comment:
Is it a module?

Reply:
every other new file has a xyz 'Module'. See client.py api.py etc